### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'oldFunc.apply') ... (line 165)

### ISSUE THREAD

**Issue by kaidjohnson** [lodash/lodash-migrate#63]
When I debug this, I'm seeing this failing on name 'join'. When I remove 'join' from `exports.unwrapped`, the failure goes away. Seems that `_.join` was added in 4.0, so I'm not sure if I'm missing something by removing it or not, but it doesn't seem to belong here.

**Comment by jdalton** [comment]
Hi @kaidjohnson!

I need more info. I can't repro the issue. It works in the browser and in Node with Lodash v3 and Lodash v4.

**Comment by kaidjohnson** [comment]
I'm running this in phantomjs 2.1.1 via karma. We have a relatively large-scale application with a lazy-loading modular architecture which makes a tool like `lodash-migrate` less useful since we would have to load every page/module manually and our code is not run on a node server. I realized that one way to ensure that it did get run over all the code was to include it in the project and run our test suite. This worked very well, aside from the failures posted here. Once I removed `join` from `exports.unwrapped`, I got all the loggy goodness that this tool provides and within minutes was able to move over to lodash v4.
